packages/fxa-graphql-api/src/decorators.ts

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { createParamDecorator, ExecutionContext } from '@nestjs/common';
+import { GqlExecutionContext } from '@nestjs/graphql';
+import { SessionTokenResult } from './auth/session-token.strategy';
+
+/**
+ * Extracts the token from an authenticated user for a GraphQL request.
+ */
+export const GqlSessionToken = createParamDecorator(
+  (data: unknown, context: ExecutionContext): string => {
```

Comment:
Nit: I don't know if it's convention or something I just picked up along the way, but `data: unknown` could be `_` if we don't intend on using it.